siiptool/scripts/subregion_capsule.py

```diff
@@ -149,9 +149,12 @@ def convert_arg_line_to_args(arg_line):
     gen_cap_cmd += ["--capflag", "PersistAcrossReset"]
     gen_cap_cmd += ["--capflag", "InitiateReset"]
     gen_cap_cmd += ["-o", args.OutputCapsuleFile]
-    gen_cap_cmd += ["--signer-private-cert", args.OpenSslSignerPrivateCertFile]
-    gen_cap_cmd += ["--other-public-cert", args.OpenSslOtherPublicCertFile]
-    gen_cap_cmd += ["--trusted-public-cert", args.OpenSslTrustedPublicCertFile]
+    if  args.OpenSslSignerPrivateCertFile:
```

Comment:
When is these files not needed. Can you have one without the other.

Reply:
Good catched. @MohamedXElGohary Can you update the change to all or nothing?